_sass/_variables.scss

```diff
@@ -61,6 +61,7 @@ $light-black: #303030;
 
 // State pages
 $green-land: #d5e5cb;
+$chart-green: #a5d78a;
```

Comment:
consider consolidating these two variable colors to stay consistent. If we set `$green-land: #a5d78a;` the land ownership map looks like this: 
<img width="412" alt="screen shot 2016-06-07 at 5 04 53 pm" src="https://cloud.githubusercontent.com/assets/4803473/15876436/09a71c5a-2cd3-11e6-9d61-c1a37d37465f.png">